src/firewall/core/fw_policy.py

```diff
@@ -833,6 +834,10 @@ def check_protocol(self, protocol):
         if not checkProtocol(protocol):
             raise FirewallError(errors.INVALID_PROTOCOL, protocol)
 
+    def check_tcp_mss_clamp(self, tcp_mss_clamp_value):
+        if not checkTcpMssClamp(tcp_mss_clamp_value):
+            raise FirewallError(errors.INVALID_RULE, tcp_mss_clamp_value)
```

Comment:
This error string could be better. Maybe something like: "tcp-mss-clamp value must be greater than or equal to 536, or the special value 'pmtu'. Invalid value '%s'" % (tcp_mss_clamp_value)